daemon/daemon.go

```diff
@@ -15,6 +15,9 @@ import (
 	"time"
 
 	"github.com/docker/libcontainer/label"
+	"github.com/docker/libnetwork"
+	"github.com/docker/libnetwork/pkg/netlabel"
```

Comment:
@mrjana Why not use the `internal` package convention (https://docs.google.com/document/d/1e8kOo3r51b2BWtTs_1uADIA5djfXhPT36s6eHVRIvaU/edit)? Should be coming with Go 1.5 but you may adopt the pattern now.

It is very odd to have a package declare that it has packages, when sub-directories of a package are already packages. These should just subdirectories.